exercises/grep/canonical-data.json

```diff
@@ -204,6 +204,37 @@
             "files": ["iliad.txt"]
           },
           "expected": []
+        },
+        {
+          "description": "One file, one match, line number and file flags",
```

Comment:
I propose that this description should explicitly say which of the two flags takes precedence.

I propose this because it makes the intent of the test easier to understand for a reader of the description.

Reply:
It makes perfect sense. I'll update the test cases with an explicit description.